### DISCUSSION TITLE
/feature/advance/customizing.html

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#12] [Announcements]
# /feature/advance/customizing.html

VanBlog 的官方网站

https://vanblog.mereith.com/feature/advance/customizing.html

**Comment by king-bcolor** [comment]
-- --- -
# 好，可以
**自定义万岁**
- - -

**Comment by king-bcolor** [comment]
# vanblog似乎没有自带的懒加载
> 我在你的演示页面发现vanblog加载的资源实际上不多，很轻
>> 但发布文章的预览图片却占了1mb

**Reply by Mereithhh** [reply]
有懒加载，但是自带的图片是没有经过压缩的，这确实是一个优化的点
